App.js

```diff
@@ -1,26 +1,79 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, { useState } from 'react';
+import styled from "styled-components"
+import {
+  Text,
+  View,
+  FlatList,
+  Alert,
+  TouchableWithoutFeedback,
+  Keyboard
+} from 'react-native';
+
+import Header from './components/Header';
+import TodoItem from './components/TodoItem';
+import AddTodo from './components/addTodo';
 
 const App = () => {
+
+  const [todos, setTodos] = useState([
+    { text: 'Buy wine', key: '1' },
+    { text: 'Learn JS', key: '2' },
+    { text: 'Yoga', key: '3' }
+  ]);
+
+  const pressHandler = (key) => {
+    setTodos(prevTodos => {
+      return prevTodos.filter(todo => todo.key != key);
+    });
+  };
+
+  const submitHandler = (text) => {
+    if (text.length > 5) {
+      setText('');
```

Comment:
you don't have access to `setText` here in this file. You need to do this reset in the AddTodo component where that state lives.